docs/references/compatibility_matrix.md

```diff
@@ -3,16 +3,17 @@
-| realtime-kernel | &#x2705;  |  &#x2705;  | &#10060;   | &#x2705;| &#x2705;| &#x2705; | &#10060; | &#10060;      | N/A              | &#x2705; | &#x2705;    |
-| ros             | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | N/A      | &#x2705;    |
-| ros-updates     | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | N/A         |
+|                 | esm-infra |  esm-apps  | livepatch  | cc-eal  |  cis    | usg      | fips     | fips-updates  | realtime-kernel  | ros      | ros-updates | anbox-cloud |
+| ----------------|:---------:|:----------:|:----------:|:-------:|:-------:|:--------:|:-----:   |:-------------:|:----------------:|:--------:|:-----------:|:-----------:|
+| esm-infra       | N/A       |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | &#x2705;    | &#x2705;    |
+| esm-apps        | &#x2705;  |  N/A       | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | &#x2705;    | &#x2705;    |
+| livepatch       | &#x2705;  |  &#x2705;  | N/A        | &#x2705;| &#x2705;| &#x2705; | &#10060; | &#10060;      | &#10060;         | &#x2705; | &#x2705;    | &#x2705;    |
+| cc-eal          | &#x2705;  |  &#x2705;  | &#x2705;   | N/A     | &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | &#x2705;    | &#x2705;    |
+| cis             | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| N/A     | &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | &#x2705;    | &#x2705;    |
+| usg             | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| N/A      | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | &#x2705;    | &#x2705;    |
+| fips            | &#x2705;  |  &#x2705;  | &#10060;   | &#x2705;| &#x2705;| &#x2705; | N/A      | &#10060;      | &#10060;         | &#x2705; | &#x2705;    | &#x2705;    |
+| fips-updates    | &#x2705;  |  &#x2705;  | &#10060;   | &#x2705;| &#x2705;| &#x2705; | &#10060; | N/A           | &#10060;         | &#x2705; | &#x2705;    | &#x2705;    |
+| realtime-kernel | &#x2705;  |  &#x2705;  | &#10060;   | &#x2705;| &#x2705;| &#x2705; | &#10060; | &#10060;      | N/A              | &#x2705; | &#x2705;    | &#x2705;    |
+| ros             | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | N/A      | &#x2705;    | &#x2705;    |
+| ros-updates     | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | N/A         | &#x2705;    |
+| anbox-cloud     | &#x2705;  |  &#x2705;  | &#x2705;   | &#x2705;| &#x2705;| &#x2705; | &#x2705; | &#x2705;      | &#x2705;         | &#x2705; | &#x2705;    | &#x2705;    |
```

Comment:
@s-makin I don't like how the table is presented after this new change. Let me know if you have a better idea on how to address that

Reply:
We had the same concerns last time and now it's come back to bite us :/ I've been digging, and I think the only way we'll be able to display this table sensibly (e.g., with vertical top headers) is by using html and css. We could use tabs, to only show one column at a time, but that would be weird. Although if we followed the grouping on the front page, that might work? The columns could then just be the columns applicable to a specific service need, and it wouldn't matter if we showed all rows for each. We might need to have a chat about it later, let me know what you think!

Reply:
Yes, I agree we should chat about this for sure